plugin/evm/vm_test.go

```diff
@@ -2819,7 +2819,7 @@ func TestRewardManagerPrecompileSetRewardAddress(t *testing.T) {
 	data, err := rewardmanager.PackSetRewardAddress(testAddr)
 	require.NoError(t, err)
 
-	gas := 21000 + 240 + rewardmanager.SetRewardAddressGasCost // 21000 for tx, 240 for tx data
+	gas := 21000 + 240 + rewardmanager.SetRewardAddressGasCost + rewardmanager.RewardAddressChangedEventGasCost // 21000 for tx, 240 for tx data
```

Comment:
why are these changed?

Reply:
because VMs will now have non-nil upgrades (if default is defined). This was running with only `subnetEVM` upgrade (Durango was nil in chainconfig) but since now we apply defaults for nils, this test is upgraded to Durango fork. In Durango precompiles have increased costs (because of events)